package.json

```diff
@@ -8,7 +8,8 @@
     "dev:cl": "cd frontend && npm run start",
     "dev:ts": "tsc -w",
     "dev:js": "nodemon dist/server.js",
-    "dev": "concurrently npm:dev:*"
+    "dev": "concurrently npm:dev:*",
+    "lint": "npx eslint --fix . && npx stylelint \"**/*.{css,scss}\""
```

Comment:
- [ ] I have question, I can see that added the `--fix` flag to fix eslint errors, but how about stylelint, should we add the same flag here after `npx stylelint \"**/*.{css,scss}\"` ?

Reply:
Hi @yonas44, This dist folder is already in .gitignore.
I'm talking about is there any way to ignore when we run the command `npm run lint`

Reply:
I am thinking of adding an ignore-pattern in the linter config file to skip specific directories